tfx/experimental/templates/taxi/README.md

```diff
@@ -24,3 +24,63 @@ about the dataset in [Google BigQuery](https://cloud.google.com/bigquery/).
 Explore the full dataset in the
 [BigQuery UI](
 https://bigquery.cloud.google.com/dataset/bigquery-public-data:chicago_taxi_trips).
+
+
+## Testing the pipeline using Stub Executors
```

Comment:
I think that this content can have a separate tutorial file, for example, under https://github.com/tensorflow/tfx/tree/master/docs/tutorials/tfx .